bigtable/google/cloud/bigtable/table.py

```diff
@@ -307,9 +307,6 @@ def read_rows(self, start_key=None, end_key=None, limit=None,
         :returns: A :class:`.PartialRowsData` convenience wrapper for consuming
                   the streamed results.
         """
-        request_pb = _create_row_request(
-            self.name, start_key=start_key, end_key=end_key, filter_=filter_,
-            limit=limit, end_inclusive=end_inclusive)
```

Comment:
@calpeyser This is problematic, you've (it seems accidentally) removed the `end_inclusive` feature.